easybuild/easyconfigs/r/RFdiffusion/RFdiffusion-1.1.0_setup.patch

```diff
@@ -0,0 +1,19 @@
+diff -ru RFdiffusion-1.1.0/setup.py RFdiffusion-1.1.0_setup/setup.py
```

Comment:
This patch already exists as `RFdiffusion-1.1.0_fix-find-packages.patch`. Please remove and use the existing one.